doc/Extensions/Smokeping.md

```diff
@@ -304,3 +304,39 @@ BASE_OPTIONS=
 ```
 
 If -B is in the list of arguments delete it.
+
+### To store smokeping rrd in librenms rrd folder
+
+```bash
+sudo systemctl stop smokeping
+sudo mv /var/lib/smokeping/ /opt/librenms/rrd/
+sudo nano /etc/smokeping/config.d/pathnames
+```
+Then update the config file:
+
+```bash
+datadir = /opt/librenms/rrd/smokeping
+dyndir = /opt/librenms/rrd/smokeping/__cgi
+```
+And give to smokeping rights to access files
+
+```bash
+sudo adduser smokeping librenms
```

Comment:
This isn't how you add a user to a group...

Reply:
@murrant I'm not sure of what you mean.
[adduser allow this use at least on Debian](https://manpages.debian.org/buster/adduser/adduser.8.en.html#Add_an_existing_user_to_an_existing_group).

Regards,
Anael

Reply:
Heh, I don't even have the adduser command on my distro.

It is probably safer to use:
usermod -a -G librenms smokeping

Reply:
Updated, thanks !